app.js

```diff
@@ -10,6 +10,7 @@ var drive = hyperdrive(db)
 var speedometer = require('speedometer')
 var prettyBytes = require('pretty-bytes')
 var explorer = require('./')
+var intro = require('intro.js')
```

Comment:
Is it a browserify magic? this is a crazy module named with `.js` under `node_modules/intro.js/`